data-raw/nwsl_data.R

```diff
@@ -0,0 +1,22 @@
+#code to create dataframe with NWSL data. 
+
+library(dplyr)
+
+nwsl_data <- data.frame(name = c("North Carolina Courage", "Utah Royals FC", "Houston Dash",
```

Comment:
Could you please put `name` on the next line and indent the rest of the lines properly? Use `Crtl-I` and it should do most of it automatically.